### PR TITLE
Feature/data attributes

### PR DESCRIPTION
**Pull request by lievenvg**
## PR Checklist

This PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] A changelog entry has been added to CHANGELOG.md if necessary

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] react application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #100 

## What is the new behavior?
Accordion now accepts `data-qa` attribute.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/accordion/src/AccordionTab/AccordionTab.spec.js [hunk 1]
```diff
@@ -44,4 +44,15 @@ describe('AccordionTab', () => {
...
+
+  test('Can contain a data-qa attribute', () => {
+    const component = mount(
+      <AccordionTab data-qa="id-1234">
+        <AccordionTabHeader />
+        <AccordionTabContent />
+      </AccordionTab>,
+    );
+
+    expect(component.prop('data-qa')).toBe('id-1234');
```
Comment (TriangleJuice) [inline packages/accordion/src/AccordionTab/AccordionTab.spec.js]:
```suggestion
    expect(component.prop('qa')).toBe('id-1234');
```

File: packages/accordion/src/AccordionTabContent/AccordionTabContent.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ type Props = {
   children: React.node
 };
 
-const AccordionTabContent = ({ children }: Props) => (
-  <div className="m-accordion__content">
+const AccordionTabContent = ({ children, "data-qa": qa }: Props) => (
```
Comment (TriangleJuice) [inline packages/accordion/src/AccordionTabContent/AccordionTabContent.js]:
```suggestion
const AccordionTabContent = ({ children, qa }: Props) => (
```

File: packages/accordion/src/AccordionTabHeader/AccordionTabHeader.js [hunk 3]
```diff
@@ -5,8 +5,8 @@ type Props = {
   children: React.node
 };
 
-const AccordionTabHeader = ({ children }: Props) => (
-  <div className="m-accordion__header">
+const AccordionTabHeader = ({ children, "data-qa": qa }: Props) => (
```
Comment (TriangleJuice) [inline packages/accordion/src/AccordionTabHeader/AccordionTabHeader.js]:
```suggestion
const AccordionTabHeader = ({ children, qa }: Props) => (
```

File: packages/accordion/src/AccordionTabHeader/AccordionTabHeader.spec.js [hunk 4]
```diff
@@ -8,4 +8,12 @@ describe('AccordionTabHeader', () => {
 
     expect(component.find('.m-accordion__header')).toBeDefined();
   });
+
+  test('Can contain a data-qa attribute', () => {
+    const component = mount(
+      <AccordionTabHeader data-qa="id-1234"/>
+    );
+
+    expect(component.prop('data-qa')).toBe('id-1234');
```
Comment (TriangleJuice) [inline packages/accordion/src/AccordionTabHeader/AccordionTabHeader.spec.js]:
```suggestion
    expect(component.prop('qa')).toBe('id-1234');
```

**Comment by lievenvg** [comment]
Unit tests updated to check for `data-qa` value.

File: packages/accordion/src/Accordion.spec.js [hunk 5]
```diff
@@ -22,4 +22,12 @@ describe('Accordion', () => {
     expect(component.find('.m-accordion__tab')).toBeDefined();
     expect(component.find('.m-accordion__tab').length).toEqual(3);
   });
+
+  test('Can contain a data-qa attribute', () => {
+    const component = shallow(
+      <Accordion qa="id-1234"/>
+    );
+
+    expect(component.find('[data-qa="id-1234"]').length).toEqual(1);
```
[the remark this answers is not in the document]
Comment (lievenvg) [reply packages/accordion/src/Accordion.spec.js]: Unit tests updated!

**Pull request merged**